legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-bigquery/src/main/java/org/finos/legend/engine/persistence/components/relational/bigquery/sql/visitor/BatchStartTimestampVisitor.java

```diff
@@ -15,11 +15,8 @@
 package org.finos.legend.engine.persistence.components.relational.bigquery.sql.visitor;
 
 import org.finos.legend.engine.persistence.components.logicalplan.values.BatchStartTimestamp;
-import org.finos.legend.engine.persistence.components.logicalplan.values.FunctionImpl;
-import org.finos.legend.engine.persistence.components.logicalplan.values.FunctionName;
-import org.finos.legend.engine.persistence.components.logicalplan.values.StringValue;
+import org.finos.legend.engine.persistence.components.logicalplan.values.DatetimeValue;
```

Comment:
If it's possible to have the same BatchStartTimestampVisitor and BatchEndTimestampVisitor for ansi and Bigquery, Let's do that.
Then we can just delete these visitors in BigQuery